src/harness/reference_models/ppa_model/ppa_ref_model.py

```diff
@@ -0,0 +1,145 @@
+#    Copyright 2016-2018 SAS Project Authors. All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+
+import multiprocessing
+
+import geojson
+import numpy as np
+from concurrent import futures
+from reference_models.geo import census_tract, utils, vincenty, nlcd
+from reference_models.propagation import wf_hybrid
+from reference_modesl.antenna import antenna
+from shapely import geometry, ops
+
+THRESHOLD_PER_10MHZ = -96
+RX_HEIGHT = 1.5
+
+census_tract_driver = census_tract.CensusTractDriver()
+nlcd_driver = nlcd.NlcdDriver()
+
+
+def _CalculateDbLossForEachPointAndGetContour(install_param, antenna_gain, latitude, longitude):
```

Comment:
Note that in general we use plural form for "a bunch of thing" and use the singular for a single objects. This makes it more easy for readability as we directly identify sequence type.

In that case, and elsewhere in the code, it would be: latitudes, longitudes

Reply:
I updated the code wherever possible and made in plural form. Let me know if I missed anywhere.